src/components/Restoration/CorrectCompanyName.vue

```diff
@@ -1,10 +1,10 @@
 <template>
```

Comment:
This is the component copied from Edit UI, then refactored here to be drop-in ready into the shared repo.

Reply:
> So, no getters or actions. Props and events only.

You're trolling me!  :-)

Reply:
This component is now in BCRS shared repo.

Reply:
This component is now in BCRS shared repo.